lib/llm.rb

```diff
@@ -28,6 +28,14 @@ def gemini(secret)
     LLM::Gemini.new(secret)
   end
 
+  ##
+  # @param host (see LLM::Ollama#initialize)
+  # @return (see LLM::Ollama#initialize)
+  def ollama(secret)
```

Comment:
What is `secret` suppose to be ? Seems to work without one ?

Reply:
@0x1eef Ollama models are deployed without a secret i think, Can this a convincing case to make secret optional too? :)

Reply:
Thanks!